examples/Cassie/input_supervisor.cc

```diff
@@ -58,6 +65,11 @@ InputSupervisor::InputSupervisor(
   switch_time_index_ = DeclareDiscreteState(1);
   prev_efforts_time_index_ = DeclareDiscreteState(1);
   prev_efforts_index_ = DeclareDiscreteState(num_actuators_);
+  soft_estop_flag_index_ = DeclareDiscreteState(1);
+
+  K_ = plant_.MakeActuationMatrix().transpose();
+  K_ *= kEStopGain;
+  std::cout << K_ << std::endl;
```

Comment:
If we're going to leave this in let's also print out "Soft E-stop Gains:" or similar

Reply:
oops meant to remove this